config.js

```diff
@@ -16,12 +16,8 @@ const DIALOG_STAGES = [
         '😱 you are late.',
     ],
     VERBAL_ALERTS = [
-        'yellow alert, meeting imminent',
-        'red alert',
-        "heyyoo... it's time",
-        'beep beep!',
-        'meeting time!',
-        'hey!! check your calendar!',
+        'spotify:track:7jL5aGalMaZRfnJ9jasABT',
```

Comment:
probably should revert

Reply:
Sure. What about migrating this into the events.json file? That would similarly expose it to the UI as per your other comment.

Reply:
good idea.